projects/extension/build.py

```diff
@@ -128,32 +128,32 @@ def install_sql(version: str | None = None) -> None:
     def install_prior_py() -> None:
         """installs the extension's python package for prior versions"""
         for version in prior_versions():
-            if os.sep in version:
-                fatal(f"'{os.sep}' in version {version}. this is not supported")
             version_target_dir = python_install_dir().joinpath(version)
             if version_target_dir.exists():
                 continue
             tmp_dir = Path(tempfile.gettempdir()).joinpath("pgai", version)
             shutil.rmtree(tmp_dir, ignore_errors=True)
```

Comment:
I think part of his suggestion was that we don't then need this line as the `finally` below will take care of it, and that the only way this folder should exist is if you created the folder manually.

Reply:
yeah, we *shouldn't need it, but it doesn't hurt to have it